app/src/components/screens/rounds/RoundsList.js

```diff
@@ -159,15 +184,24 @@ class RoundsList extends React.Component {
     return (
       <FlatList
         data={roundsToRender}
-        renderItem={({ item }) => (
-          <RoundListItem
-            detail={this.roundItemPress}
-            onDeleteStoredRound={this.onDeleteStoredRound}
-            {...item}
-            auth={auth}
-            pending
-          />
-        )}
+        renderItem={({ item, index }) =>
+          item.start !== undefined ? (
```

Comment:
```suggestion
          !!item.start ? (
```
Tiene que ser si o si `undefined`? Que pasa si es `null`?